script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -584,7 +584,7 @@ You think you're fine just because you didn't faint?
 -- Page 470, Offset 17741.
 -- あまり視えないものを視ようとしないで。そんなに脳を酷使したら廃人になりかねないわ」
 You shouldn't strain yourself to make out things you can't see.
- If you abuse your brain like that, you run the risk of turning into a cripple."
+ If you abuse your brain like that, you run the risk of turning into a cripple." 
```

Comment:
Try and avoid adding these trailing spaces, re-exporting the script using luna_cli is a way to ensure everything is formatted correctly.

Reply:
this was marked resolved but the trailing space is still here.